addons/mail/static/src/core/common/message.js

```diff
@@ -230,7 +230,12 @@ export class Message extends Component {
         ) {
             return url("/mail/static/src/img/email_icon.png");
         }
-        return this.message.author.avatarUrl;
+
+        if (this.message.author) {
+            return this.message.author.avatarUrl;
+        }
+
+        return '/mail/static/src/img/smiley/avatar.jpg'
```

Comment:
Better use `DEFAULT_AVATAR` (https://github.com/odoo/odoo/blob/saas-17.1/addons/mail/static/src/core/common/persona_service.js#L9) instead of a hard-coded string.